templates/README.md

```diff
@@ -0,0 +1,11 @@
+
+# Package templates
+
+> This folder contains a bunch of packages that
+> act as templates when creating new packages for specific
+> purposes.
+
+The current templates are:
+- frontend-package (package with a JS/TS/React.js/JEST test setup)
+
+When creating new packages from the templates, simply copy the folder, rename the package in the package.json and you're good to go.
```

Comment:
Yes, but ideally this would be a script to enforce consistency. You do not "simply" copy a folder ;)